OracleWebLogic/samples/12212-msiserver/container-scripts/launcher.sh

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/bash
```

Comment:
This `launcher.sh` script has no handlers for stop/start. You need to add handlers so that users can use `docker stop` effectively.